src/pudl_archiver/metadata/sources.py

```diff
@@ -287,4 +287,114 @@
         "license_pudl": LICENSES["cc-by-4.0"],
         "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
     },
+    "eccoalclosure": {
+        "title": "Coal Closure Energy Communities Dataset",
+        "path": "https://edx.netl.doe.gov/dataset/ira-energy-community-data-layers",
+        "description": (
+            "This dataset outlines energy communities across the U.S. based off part of the Inflation "
+            "Reduction Act’s (IRA) definition that is focused on census tracts and directly adjoining "
+            "tracts that have had coal mine closures since 1999 or coal-fired electric generating unit "
+            "retirements since 2009."
+        ),
+        "working_partitions": {"years": [2024]},
+        "keywords": sorted(
+            {
+                "energy",
+                "community",
+                "coal",
+                "mines",
+                "closure",
+                "ira",
+            }
+        ),
+        "license_raw": LICENSES["us-govt"],
+        "license_pudl": LICENSES["cc-by-4.0"],
+        "contributors": [CONTRIBUTORS["catalyst-cooperative"]],
+    },
+    "ecffemployment": {
+        "title": "MSA/Non-MSAs that only meets the Fossil Fuel Employment 0.17% Threshold Dataset",
```

Comment:
can you make this title also follow the convention of "SOURCE NAME -- Full title"